src/test/java/org/ibp/api/rest/preset/PresetResourceTest.java

```diff
@@ -71,8 +71,8 @@ public void setup() throws Exception {
 		this.toolId = 23;
 
 		this.type = PresetType.LABEL_PRINTING_PRESET.getName();
-		this.selectedField = Arrays.asList(Arrays.asList(4, 13));
-		this.barcodeSetting = new LabelPrintingPresetDTO.BarcodeSetting(Boolean.TRUE, Boolean.FALSE, Arrays.asList(2));
+		this.selectedField = Arrays.asList(Arrays.asList("STATIC_4", "STATIC_13"));
```

Comment:
@cuenyad no test for the new enum FieldType.NAME

Reply:
done!